build.gradle.kts

```diff
@@ -4,10 +4,10 @@ plugins {
 
 repositories {
```

Comment:
The template plugin uses local > runelite repo > maven central https://github.com/runelite/plugin-hub/blob/master/templateplugin/build.gradle#L5-L11
does changing this only change things in our local build environment and CI?

Reply:
i made that change due to https://discord.com/channels/301497432909414422/419891709883973642/1240823360242581534
but we can revert since the ratelimit was increased

Reply:
I think we can leave this change out until it happens again